test/docs.jl

```diff
@@ -30,7 +30,7 @@
         #pretty print the model to the terminal
         #print(pm.model)
 
-        @test num_nonlinear_constraints(pm.model) == 12
+        @test JuMP.num_nonlinear_constraints(pm.model) == 0
```

Comment:
I _think_ this change is good? Linear/quadratic constraints are now inferred as linear/quadratic.

Reply:
This looks like a bug to me, `ACPPowerModel` should include some constraints with cos and sin that will be nonlinear.

Reply:
D'oh. The proper change is below:
```Julia
@test JuMP.num_constraints(pm.model, JuMP.NonlinearExpr, JuMP.MOI.EqualTo{Float64}) == 12
```